g3/main.py

```diff
@@ -13,6 +16,21 @@ class MessageTone(str, Enum):
     FUNNY = "funny"
 
 
+@app.command()
+def configure() -> None:
```

Comment:
Can we add a hook in `__main__` too so that we implicitly call `configure` in case `~/.g3/config` is missing (i.e. during first execution)?

Reply:
For the time being, I added a check that instructs the user to call `g3 configure` explicitly.